### PR TITLE
The new new group

### PR DESCRIPTION
**Pull request by Bathande**

### CODE REVIEW

File: skietbaan_fe/src/components/AddMembersGroup.js [hunk 1]
```diff
@@ -126,10 +127,11 @@ class AddMembersGroup extends Component {
   }
 
   render() {
+    console.log("render",this.props.memberIds)
```
Comment (DinolanNaidoo) [inline skietbaan_fe/src/components/AddMembersGroup.js]: console log

File: skietbaan_fe/src/components/EditGroup.js [hunk 2]
```diff
@@ -153,6 +152,7 @@ class EditGroup extends Component {
 								!this.state.filterText ||
 								post.username.toLowerCase().startsWith(this.state.filterText.toLowerCase()) ||
 								post.email.toLowerCase().startsWith(this.state.filterText.toLowerCase())
+
```
Comment (JurieOberholzer) [inline skietbaan_fe/src/components/EditGroup.js]: remove white space

File: skietbaan_fe/src/components/createPages.js [hunk 3]
```diff
@@ -41,6 +42,7 @@ export class createPages extends Component {
 	updateCreateContainer() {
 		switch (this.selectedButton) {
 			case 1:
+
```
Comment (JurieOberholzer) [inline skietbaan_fe/src/components/createPages.js]: remove white space

File: skietbaan_fe/src/components/createPages.js [hunk 4]
```diff
@@ -69,34 +71,43 @@ export class createPages extends Component {
...
+		this.props.pageState(10);
 		this.props.compSelectedPages(1);
 		this.setState({ selectedButtonCreateViewGroups: 1 });
 	}
 
 	viewGroups() {
+		this.props.pageState(0);
 		this.props.compSelectedPages(2);
 		this.setState({ selectedButtonCreateViewGroups: 2 });
+
```
Comment (JurieOberholzer) [inline skietbaan_fe/src/components/createPages.js]: ditto

File: skietbaan_fe/src/components/AddGroup.js [hunk 5]
```diff
@@ -42,7 +41,8 @@ class AddGroup extends Component {
     if (this.state.groups.indexOf(this.state.name.toLowerCase()) === -1) {
       if (this.state.name.length != 0) {
         this.props.getName(this.state.name);
-        history.push("/Groups");
+       history.push("/Groups");
```
Comment (JurieOberholzer) [inline skietbaan_fe/src/components/AddGroup.js]: indentation?

File: skietbaan_fe/src/components/createPages.js [hunk 6]
```diff
@@ -69,34 +70,40 @@ export class createPages extends Component {
...
+		this.props.pageState(10);
 		this.props.compSelectedPages(1);
 		this.setState({ selectedButtonCreateViewGroups: 1 });
 	}
 
 	viewGroups() {
+		this.props.pageState(0);
 		this.props.compSelectedPages(2);
-		this.setState({ selectedButtonCreateViewGroups: 2 });
+		this.setState({ selectedButtonCreateViewGroups: 2 });
```
Comment (JurieOberholzer) [inline skietbaan_fe/src/components/createPages.js]: remove white space at the end

File: skietbaan_fe/src/components/EditGroup.js [hunk 7]
```diff
@@ -152,7 +151,7 @@ class EditGroup extends Component {
 							return (
 								!this.state.filterText ||
 								post.username.toLowerCase().startsWith(this.state.filterText.toLowerCase()) ||
-								post.email.toLowerCase().startsWith(this.state.filterText.toLowerCase())
+								post.email.toLowerCase().startsWith(this.state.filterText.toLowerCase())
```
Comment (JurieOberholzer) [inline skietbaan_fe/src/components/EditGroup.js]: remove white space behind

**Pull request merged**